pkg/api/routes.go

```diff
@@ -97,7 +97,11 @@ func (rh *RouteHandler) SetupRoutes() {
 	rh.c.Router.PathPrefix("/swagger/v2/").Methods("GET").Handler(httpSwagger.WrapHandler)
 	// Setup Extensions Routes
 	if rh.c.Config != nil && rh.c.Config.Extensions != nil {
```

Comment:
also check rh.c.Config.Extensions.Search.Enable

Reply:
This code was agnostic of what extensions we have implemented. Do we want it to be aware of what extensions we have?
The check for the extensions themselves just enabled the graphql API. And what the API can provide is controlled from inside ext.SetupRoutes.

I believe we should leave this code as is was on master, and modify  the code in https://github.com/anuvu/zot/pull/172/files#diff-86b8c7ae55e63b332b1b61b1e50093ca23babbdf0ad78ff1f74bcbec501cc188R58 to read this configuration.
The configuration is already available in rh.c.Config.Extensions, we don't need to pass it as a separate parameter.

Reply:
Sure, I will add a check there